.github/workflows/_run_test.yml

```diff
@@ -0,0 +1,102 @@
+
+        - id: main
+          name: Run main script
+          timeout-minutes: ${{ inputs.TIMEOUT }}
+          run: |
+            nvidia-smi
+            mkdir -p ${{ github.run_id }}
+            cd ${{ github.run_id }}/
+            set +e
+            (
+              set -e
+
+              cmd=$(cat <<"RUN_TEST_EOF"
+            # Sanity check the driver/cuda combo
+            cudaCheck
+            # In case git commands need to be run inside Aligner
+            git config --global --add safe.directory $ALIGNER_REPO_DIR
+            ${{ inputs.SCRIPT }}
+            RUN_TEST_EOF
+            )
+              #docker run --rm --runtime=nvidia --gpus all --shm-size=8g --env TRANSFORMERS_OFFLINE=0 --env HYDRA_FULL_ERROR=1 --volume /mnt/datadrive/TestData:/home/TestData:ro nemoci.azurecr.io/nemo_aligner_container:${{ github.run_id }} bash -eux -o pipefail -c "$cmd"
+              docker run --rm --runtime=nvidia --gpus all --shm-size=8g \
+                --env TRANSFORMERS_OFFLINE=0 --env HYDRA_FULL_ERROR=1 --env HF_HOME=/home/TestData/aligner/hf_home --env ALIGNER_CI_DIR=/home/TestData/aligner --env ALIGNER_REPO_DIR=/opt/NeMo-Aligner \
+                --volume /mnt/datadrive/TestData/aligner/checkpoints:/home/TestData/aligner/checkpoints:ro \
+                --volume /mnt/datadrive/TestData/aligner/hf_home/hub:/home/TestData/aligner/hf_home/hub:ro \
+                nemoci.azurecr.io/nemo_aligner_container:${{ github.event.pull_request.number }} \
+                bash -eux -o pipefail -c "$cmd"
+            ) 2> >(tee err.log)
+
+            EXIT_CODE=$?
+
+            echo "log=$(tail -c 2000 err.log |  base64 -w 0)" >> "$GITHUB_OUTPUT"
+            
+            exit $EXIT_CODE
+
+        - uses: "NVIDIA/NeMo/.github/actions/cancel-workflow@main"
+          if: failure() && inputs.IS_OPTIONAL == false
+        - name: after_script
+          if: always() && inputs.AFTER_SCRIPT != ':'
+          run: |
+            cmd=$(cat <<"RUN_TEST_EOF"
+            ${{ inputs.AFTER_SCRIPT }}
+            RUN_TEST_EOF
+            )
+            #docker run --rm --runtime=nvidia --gpus all --shm-size=8g --env TRANSFORMERS_OFFLINE=0 --env HYDRA_FULL_ERROR=1 --volume /mnt/datadrive/TestData:/home/TestData nemoci.azurecr.io/nemo_aligner_container:${{ github.run_id }} bash -eux -o pipefail -c "$cmd"
+            docker run --rm --runtime=nvidia --gpus all --shm-size=8g \
+              --env TRANSFORMERS_OFFLINE=0 --env HYDRA_FULL_ERROR=1 --env HF_HOME=/home/TestData/aligner/hf_home --env ALIGNER_CI_DIR=/home/TestData/aligner \
+              --volume /mnt/datadrive/TestData/aligner/checkpoints:/home/TestData/aligner/checkpoints:ro \
+              --volume /mnt/datadrive/TestData/aligner/hf_home/hub:/home/TestData/aligner/hf_home/hub:ro \
+              nemoci.azurecr.io/nemo_aligner_container:${{ github.event.pull_request.number }} \
+              bash -eux -o pipefail -c "$cmd"
```

Comment:
https://github.com/NVIDIA/NeMo/blob/main/.github/workflows/_test_template.yml#L54

Reply:
addressed in https://github.com/NVIDIA/NeMo-Aligner/pull/356